package.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "nomios-web",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "start": "craco start",
+    "build": "craco build",
+    "test": "exit 0",
+    "lint": "eslint --ignore-path .gitignore . && stylelint --ignore-path .gitignore \"**/*.css\"",
+    "prerelease": "npm t && npm run lint",
+    "release": "standard-version",
+    "eject": "react-scripts eject"
+  },
+  "dependencies": {
+    "@craco/craco": "^3.5.0",
+    "@jackwilsdon/craco-use-babelrc": "^1.0.0",
+    "react": "^16.8.4",
+    "react-dom": "^16.8.4",
+    "react-scripts": "2.1.8"
+  },
+  "devDependencies": {
+    "@commitlint/config-conventional": "^7.5.0",
+    "babel-preset-moxy": "^3.0.4",
+    "commitlint": "^7.5.2",
+    "eslint-config-moxy": "^7.0.1",
+    "husky": "^1.3.1",
+    "lint-staged": "^8.1.5",
+    "postcss-preset-moxy": "^4.0.0",
+    "standard-version": "^5.0.1",
+    "stylelint": "^9.10.1",
+    "stylelint-config-moxy": "^5.0.0"
+  },
+  "browserslist": [
```

Comment:
We don't need this as the `browsers` support is pre-defined in our babel/postcss configs.